Source/RapyutaSimulationPlugins/Private/Core/RRNetworkPlayerController.cpp

```diff
@@ -88,9 +88,10 @@ void ARRNetworkPlayerController::InitSimStateClientROS2()
     // Init SimStateClient's [ROS2Node] & [ClockPublisher]
     UWorld* currentWorld = GetWorld();
     SimStateClientROS2Node = currentWorld->SpawnActor<AROS2Node>();
+    SimStateClientROS2Node->AttachToActor(this, FAttachmentTransformRules::KeepRelativeTransform);
```

Comment:
We may need to switch `AROS2Node` to an Actor component or just UObject then since it seems to have no Actor/Pawn functionalitites.

Reply:
We may need to switch `AROS2Node` to an Actor component or just UObject then since it seems to have no Actor/Pawn functionalities.